cnf-tests/hack/common.sh

```diff
@@ -2,8 +2,7 @@
 
 set -e
 
-pushd .
-cd "$(dirname "$0")/.."
+pushd "$(dirname "$0")/.."
```

Comment:
@yuvalk : I wasn't sure if this was okay... Was there a reason for doing `pushd .; cd foo` instead of just `pushd foo`?  I find the latter much clearer to debug.

Reply:
I'm ok with your fix here.
just worried that the commit msg hook didnt catch it.

ah it's cnf-tests/hack/common.sh and not hack/common.sh (which have the same issue)
actually IDK why we have 2.
I'd prefer cnf-tests/hack/* to include `hack/common.sh`
if possible
@fedepaol

Reply:
hmm actually, that'll require special treat for exactly that pushd :-/

Reply:
If you did `$(basename ${BASH_SOURCE[-1]})` instead of `$(basename $0)` maybe that would solve it?

Reply:
On the other hand, maybe having these sub-project hack dirs being completely independent may be wise if/when we repackage either of them separately?